databend-jdbc/src/main/java/com/databend/jdbc/DatabendConnection.java

```diff
@@ -1,24 +1,17 @@
 package com.databend.jdbc;
 
-import com.databend.client.ClientSettings;
-import com.databend.client.DatabendClient;
-import com.databend.client.DatabendClientV1;
-import com.databend.client.DatabendSession;
-import com.databend.client.PaginationOptions;
-import com.databend.client.QueryRequest;
-import com.databend.client.StageAttachment;
+import com.databend.client.*;
```

Comment:
import com.xx.xx.* is not a good practice, maybe we should just  import what we use.

Reply:
fixed.
Intellij auto change it when  adding Import